MyOnboarding/test/MyOnboardingApp.Api.Tests/TodoListControllerTest.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using MyOnboardingApp.Api.Controllers;
+using MyOnboardingApp.Api.Models;
+using NUnit.Framework;
+
+namespace MyOnboardingApp.Tests
+{
+    [TestFixture]
+    public class TodoListControllerTest
+    {
+        private readonly TodoListController _controller = new TodoListController();
+        public TodoListItem TestItem { get; set; }
+
+        [SetUp]
+        public void SetUp()
+        {
+            TestItem = new TodoListItem{ Text = "coffee first" };
+            TodoListController.Items.Add(TestItem);
```

Comment:
Do not access TodoListController.Items at all in tests, just test that the test returns correct StatusCode. Update all tests.